src/printer/index.ts

```diff
@@ -304,7 +305,7 @@ function splitTextToDocs(node: TextNode): Doc[] {
 
 	const textLines = text.split(/[\t\n\f\r ]+/);
 
-	let docs = join(line, textLines).parts.filter((s) => s !== '');
+	let docs = join(line, textLines).filter((s) => s !== '');
```

Comment:
I guess they change what `join`  returns

Reply:
join returns "concat", which is "array" in  V3.